doc/nrf/releases/release-notes-latest.rst

```diff
@@ -12,6 +12,29 @@ Changelog
 
 See the `master branch section in the changelog`_ for a list of the most important changes.
 
+
+sdk-mcuboot
+===========
+
+The MCUboot fork in |NCS| contains all commits from the upstream MCUboot repository up to and including ``???``, plus some |NCS| specific additions.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+  * Fixed an issue where a left-over image trailer was sometimes not erased after reset.
```

Comment:
Did I interpret the input correctly?

Reply:
@ru-fu This fixes an issue where an image might be erased, but a trailer left behind.

Reply:
```suggestion
  * Fixed an issue where after erasing an image, an image trailer might be left behind.
```